src/tcgwars/logic/impl/gen8/RebelClash.groovy

```diff
@@ -2410,21 +2410,22 @@ public enum RebelClash implements LogicCardInfo {
           attackRequirement {}
           onAttack {
             damage 180
+            cantUseAttack(thisMove, self, 10000)
```

Comment:
We can remove the commented out text here, it'll still be in git history if we need it

Reply:
Maybe add a TODO tag instead